EasyRestProjectSpecflow/PageObjects/CurrentOrdersPage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium;
+using SpecFlow.Actions.Selenium;
+
+namespace SpecFlowProject.Pages
+{
+    public class CurrentOrdersPage
+    {
+
+        private readonly IBrowserInteractions _browserInteractions;
+
+        public CurrentOrdersPage(IBrowserInteractions browserInteractions)
+        {
+            _browserInteractions = browserInteractions;
+        }
+        private IWebElement _currentOrdersButton => _browserInteractions.WaitAndReturnElement
+            (By.XPath("//span[contains(text(), 'Current Orders')]"));
+
+        private IWebElement _waitingforconfirm => _browserInteractions.WaitAndReturnElement
```

Comment:
Better use camel case for " waitingforconfirm "element

Reply:
thank you for your comments)